template/template.go

```diff
@@ -238,6 +238,20 @@ func NewTemplateExpander(text string, name string, data interface{}, timestamp c
 				}
 				return fmt.Sprintf("%.4g%ss", v, prefix)
 			},
+			"humanizeTimestamp": func(v interface{}) string {
+				switch ts := v.(type) {
```

Comment:
I don't get the value via `query`. It's a `clientmodel.Timestamp` that I get passed into my template for /alerts. I could convert that to float64, but that seems ugly. I can haz automatic input type detection here?